pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -268,15 +267,6 @@ func main() {
 
 	%s
 
-	if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
-		setupLog.Error(err, "unable to set up health check")
-		os.Exit(1)
-	}
-	if err := mgr.AddReadyzCheck("check", healthz.Ping); err != nil {
-		setupLog.Error(err, "unable to set up ready check")
-		os.Exit(1)
-	}
-
```

Comment:
You can't remove these because the configured health and ready checks in the manager will fail (no checks are being served). The names should be updated to "healthz" and "readyz" for proper mapping to their endpoint names though.

Reply:
Ops .. crazyness thoughts.tks